plenum/test/batching_3pc/catch-up/test_state_reverted_before_catchup.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from plenum.common.constants import DOMAIN_LEDGER_ID
+from plenum.test.batching_3pc.helper import send_and_check, \
+    add_txns_to_ledger_before_order, start_precatchup_before_order
+from plenum.test.delayers import cDelay
+from plenum.test.helper import signed_random_requests, sendRandomRequests, waitForSufficientRepliesForRequests
+from plenum.test.node_catchup.helper import waitNodeDataUnequality
+from plenum.test.test_node import getNonPrimaryReplicas
+
+@pytest.fixture(scope="module")
+def tconf(tconf, request):
+    oldSize = tconf.Max3PCBatchSize
+    tconf.Max3PCBatchSize = 10
+
+    def reset():
+        tconf.Max3PCBatchSize = oldSize
+
+    request.addfinalizer(reset)
+    return tconf
+
+def test_unordered_state_reverted_before_catchup(tconf, looper, txnPoolNodeSet, client,
+                            wallet1):
+    # CONFIG
+
+    ledger_id = DOMAIN_LEDGER_ID
+    non_primary_replica_1 = getNonPrimaryReplicas(txnPoolNodeSet, instId=0)[0]
+    non_primary_node_1 = non_primary_replica_1.node
+
+    # send reqs and make sure we are at the same state
+    reqs = signed_random_requests(wallet1, tconf.Max3PCBatchSize)
```

Comment:
what's wrong with sending a 5/10 reqs, is there a need to send requests equal to batch size